<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CCSI UQ/Opt Framework - Preview Inputs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="table">
       <property name="minimumSize">
        <size>
         <width>371</width>
         <height>0</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Input 1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="fixedInputsCheck">
         <property name="text">
          <string>View Fixed Inputs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="group1D">
     <property name="title">
      <string>Plots</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select Inputs to Graph
 (Shift+Click or Ctrl+Click to select Multiple):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="inputList">
          <property name="toolTip">
           <string>Select single or multiple input parameter(s)
to view 1-D scatterplot(s) of the ensemble.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="graph1DButton">
          <property name="text">
           <string>Graph 1-D Scatter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="graph2DScatterButton">
          <property name="text">
           <string>Graph 2-D Scatter (2 Inputs Only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="graph2DDistButton">
          <property name="text">
           <string>Graph Distribution</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>248</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>218</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
